src/dlgConnectionProfiles.cpp

```diff
@@ -1485,6 +1485,7 @@ bool dlgConnectionProfiles::copyProfileWidget(QString& profile_name, QString& ol
 
     // add the new widget in
     profiles_tree_widget->addItem(pItem);
+    pItem->setIcon(customIcon(profile_name, std::nullopt));
```

Comment:
```suggestion
    pItem->setIcon(customIcon(profile_name, nullptr));
```

Reply:
why?

Reply:
`std::nullopt` is just for `std::optional`. The standard way to represent an an argument to a place that takes a point is to give a null pointer - ie `nullptr`. Check out https://docs.microsoft.com/en-us/cpp/extensions/nullptr-cpp-component-extensions?view=msvc-160 for example

Reply:
but the second argument is `std::optional`: `customIcon(const QString& text, const std::optional<QColor>& backgroundColor)`

Reply:
Yes it is, I misread 👍